content/docs/platform/developer/webhooks/inbound-webhooks.mdx

```diff
@@ -0,0 +1,124 @@
+---
+pageTitle: 'Inbound Webhooks'
+title: 'Inbound Webhooks'
+description: 'Learn how to configure inbound webhooks to receive real-time notification statuses from third-party providers.'
+icon: 'Download'
+---
+
+Inbound webhooks let Novu receive and process real-time events sent by third-party providers such as SendGrid, Amazon SES, Mailgun, and Resend. These events describe what happens to notifications after they are sent by Novu, such as, when an email is delivered, opened, clicked, or bounced.
+
+When Novu receives an inbound webhook event, it:
+- Parses and normalizes the provider’s event into Novu’s internal status model.
+- Updates the message entity and step-run trace, so the event is visible in the activity feed.
+- Forwards supported events to outbound webhooks if they are enabled in your project.
+- Enriches the notification lifecycle with engagement and delivery data, which can also power analytics and workflow variables.
+
+With inbound webhooks enabled, you get a full picture of the notification lifecycle, not just that a message was “sent,” but whether it reached the recipient, was opened, or engaged with. This closes the feedback loop and allows you to build more reliable notification experiences.
+
+## How to enable the inbound webhook
+
+First [create the integration](/platform/integrations/email/adding-email). After that, you can enable inbound webhooks by following these steps:
+1. Navigate to the Integrations Store page from the sidebar of your Novu dashboard.
+2. Click **Connect Provider** and select the email provider you want to configure.
+3. Scroll down to find the **Email Activity Tracking** section and click the toggle to enable it.
+  ![Enable the inbound webhook](/images/developer-tools/enable-inbound.png)
+
+When enabled, Novu will auto-configure delivery webhooks using your existing API key.
+![Inbound webhook enabled](/images/developer-tools/inbound-enabled.png)
+
+Inbound webhooks connect your provider’s event stream with Novu’s event processing system. Once enabled, Novu automatically creates and manages webhook endpoints with supported providers (such as SendGrid or Mailgun) so that delivery and engagement events can flow back into your Novu project.
```

Comment:
```suggestion
Inbound webhooks connect your provider’s event stream with Novu’s event processing system. Once enabled, Novu automatically creates and manages webhook endpoints with supported providers (such as SendGrid or Mailgun <we should either remove those provider or add the other supported ones like SES and Resent>) so that delivery and engagement events can flow back into your Novu project.
```